tsuru/client/init.go

```diff
@@ -0,0 +1,166 @@
+package client
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/tsuru/tsuru/cmd"
+)
+
+type Init struct{}
+
+func (i *Init) Info() *cmd.Info {
+	return &cmd.Info{
+		Name:  "init",
+		Usage: "init",
+		Desc: `
+Creates a standard example of .tsuruignore , tsuru.yaml and Procfile 
+	on the current project directory.
+
+"Procfile" describes the components required to run an application. 
+	It is the way to tell tsuru how to run your applications;
+
+".tsuruignore" describes to tsuru what it should not add into your 
+	deploy process, via "tsuru app-deploy" command and works just 
```

Comment:
I'd remove this sentence: `works just like you're used to use .gitignore`. You can use `.tsuruignore` to avoid sending files that were commited, but aren't necessary for running the app, like tests and documentation